website/docs/guides/best-practices/materializations/materializations-guide-2-available-materializations.md

```diff
@@ -0,0 +1,57 @@
+displayText: Materializations best practices
+hoverSnippet: Learn how to utilize materializations in dbt.
+---
+
+Views and tables and incremental models, oh my! In this section we’ll start getting our hands dirty digging into the three basic materializations that ship with dbt. They are considerably less scary and more helpful than lions, tigers, or bears — although perhaps not as cute (can data be cute? We at dbt Labs think so). We’re going to define, implement, and explore:
+
+- 🔍 **views**
+- ⚒️ **tables**
+- 📚 **incremental model**
+
+:::info
+👻 There is a fourth default materialization available in dbt called **ephemeral materialization**. It is less broadly applicable than the other three, and better deployed for specific use cases that require weighing some tradeoffs. We chose to leave it out of this guide and focus on the three materializations that will power 99% of your modeling needs.
+:::
+
+**Views and Tables are the two basic categories** of object that we can create across warehouses. They exist natively as types of objects in the warehouse, as you can see from this screenshot of Snowflake (depending on your warehouse the interface will look a little different). **Incremental models** and other materializations types are a little bit different. They tell dbt to **construct tables in a special way**.
+
+![Tables and views in the browser on Snowflake.](/img/guides/best-practices/materializations/tables-and-views.png)
+
+### Views
+
+- ✅ **The default materialization in dbt**. A starting project has no configurations defined for materializations, which means _everything_ is by default built as a view.
+- 👩‍💻 **Store _only the SQL logic_ of the transformation in the warehouse, _not the data_**. As such, they make a great default. They build almost instantly and cost almost nothing to build.
+- ⏱️ Always reflect the **most up-to-date** version of the input data, as they’re run freshly every time they’re queried.
+- 👎 **Have to be processed every time they’re queried, so slower to return results than a table of the same data.** That also means they can cost more over time, especially if they contain intensive transformations and are queried often.
+
+### Tables
+
+- 🏗️ **Tables store the data itself** as opposed to views which store the query logic. This means we can pack all of the transformation compute into a single run. A view is storing a _query_ in the warehouse. Even to preview that data we have to query it. A table is storing the literal rows and columns on disk.
+- 🏎️ Querying lets us **access that transformed data directly**, so we get better performance. Tables feel **faster and more responsive** compared to views of the same logic.
+- 💸 **Improves compute costs.** Compute is significantly more expensive than storage. So while tables use much more storage, it’s generally an economical tradeoff, as you only pay for the transformation compute when you build a table during a job, rather than every time you query it.
```

Comment:
>Compute is significantly more expensive than storage.

This is true for platforms that separate the two (❄️), but not necessarily true for traditional Redshift where you pay for the time the node is spun up regardless of what it's doing, and have a fixed storage capacity per node. I don't know exactly how it shakes out for BQ but wonder whether this is good advice there but for the wrong reason?